src/features/landing/components/Header/HeaderDrawerSocials/HeaderDrawerSocials.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from '@mui/material';
+import Image from 'next/image';
+
+import CustomLink from '@/components/ui/CustomLink/CustomLink';
+
+const SOCIALS = [
+  { id: 'instagram', url: '/assets/svg/header-drawer/socials/instagram.svg', href: '/' },
+  { id: 'github', url: '/assets/svg/header-drawer/socials/github.svg', href: '/' },
+  { id: 'facebook', url: '/assets/svg/header-drawer/socials/facebook.svg', href: '/' },
+  { id: 'linked-in', url: '/assets/svg/header-drawer/socials/linked-in.svg', href: '/' },
+];
+
+export default function HeaderDrawerSocials() {
```

Comment:
its duplicated code, you already have component like this, read about DRY and after refactor this code